include/utility.h

```diff
@@ -29,13 +32,13 @@ struct IpAddress {
     IpAddress(const char* host, const char* port);
 };
 
-std::string serializeIpAddress(const IpAddress& ipAddress);
-IpAddress deserializeIpAddress(const std::string& jsonString);
+utility::IpAddress* serializeIpAddressToProto(const IpAddress& ipAddress);
```

Comment:
Same question here.

Reply:
https://github.com/DCP-CloudMesh/CloudMesh/pull/7/files#diff-02a2971fb3961e26e4d94fdc7b4dd2d48e887971f46d0f64127e5bef20814799R15

Looks like it implements deep copy rather than serialization

Reply:
see above!